include/model.h

```diff
@@ -160,9 +136,16 @@ struct Tensor {
     part = LogicalPartition::NO_PART;
     part_grad = LogicalPartition::NO_PART;
   }
+  void inline_map(FFConfig &config);
+  void inline_unmap(FFConfig &config);
+  template<typename T>
+  T* get_raw_ptr(FFConfig &config);
+  void attach_raw_ptr(FFConfig &config, void *raw_ptr, bool column_major);
+  void detach_raw_ptr(FFConfig &config);
   int numDim, adim[MAX_DIM], pdim[MAX_DIM];
   LogicalRegion region, region_grad;
   LogicalPartition part, part_grad;
+  PhysicalRegion physical_region;
```

Comment:
It is used for doing an inline mapping of a tensor like this
tensor.inline_map()
np_array = tensor.get_array()
do whatever operations with the np_array
tensor.inline_unmap()

It allows interoperation between a tensor and Numpy array. But I am not sure if this feature is useful or not.

Reply:
After looking through the entire pull request, I think the `physical_region` makes sense now. It will be useful for supporting imperative DL runtimes (e.g., PyTorch or TensorFlow Eager).